src/Untagged/TypeCheck.purs

```diff
@@ -11,17 +11,17 @@ import Prelude
 
 import Data.Maybe (Maybe(..))
 import Data.Newtype (class Newtype)
-import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+import Data.Symbol (class IsSymbol, reflectSymbol)
 import Foreign (Foreign, isNull, typeOf, unsafeToForeign)
 import Foreign.Object (Object)
 import Foreign.Object as Object
 import Literals.Null (Null)
 import Literals.Undefined (Undefined)
-import Prim.RowList (class RowToList, Cons, Nil, kind RowList)
+import Prim.RowList (class RowToList, Cons, Nil, RowList)
 import Type.Proxy (Proxy(..))
-import Type.RowList (RLProxy(..))
 import Unsafe.Coerce (unsafeCoerce)
 
+class HasRuntimeType :: forall k. k -> Constraint
 class HasRuntimeType a where
   hasRuntimeType :: Proxy a -> Foreign -> Boolean
```

Comment:
Was this already `Proxy a` in PS `v0.13.x`?

Reply:
Yup this was proxy in 0.13.

Basically --

```
ps: hasRuntimeType (Proxy :: _ String) someForeign
js: typeof someForeign === "String"
```

As pointed out by @thomashoneyman though, these only make sense for `Type`. So perhaps fixing them to `Type` would be best.